.github/workflows/prbuild.yaml

```diff
@@ -17,6 +17,18 @@ jobs:
       - name: integration tests
         run: |
           make integration
+  # TODO: re-enable once Ingress v1 support is complete
```

Comment:
So just to make sure I understand, this adds the tests but doesn't execute unless you do it manually? Is there an easy way to comment out the tests that will always fail / or are not yet supported?

Reply:
> So just to make sure I understand, this adds the tests but doesn't execute unless you do it manually?

yep, there is an `--e2e-focus` flag available for `sonobuoy`, testing that out

Reply:
so far it doesn't look like that flag is respected, also looking through the Ingress conformance code I don't see anywhere it grabs the `E2E_FOCUS|SKIP` variables, doesn't have a flag for that either

from the `sonobuoy run` help

```
--e2e-focus string                                         Specify the E2E_FOCUS flag to the conformance tests. Overrides --mode. (default "\\[Conformance\\]")
--e2e-skip string                                          Specify the E2E_SKIP flag to the conformance tests. Overrides --mode. (default "\\[Disruptive\\]|NoExecuteTaintManager")
```

Reply:
Also there is only one actual `go` test named `TestSuite` so using go filtering (`-test.run` flag) doesn't work

Reply:
Using `cucumber` scenario tags works, but that logic isn't piped through the e2e script/docker image etc., would have to do some more plumbing

Added https://github.com/kubernetes-sigs/ingress-controller-conformance/issues/86